internal/pkg/config/testdata/config/full.yaml

```diff
@@ -42,14 +25,30 @@ trust_zones:
             - tz1
       jwt_issuer: https://tz2.example.com
       bundle_endpoint_profile: BUNDLE_ENDPOINT_PROFILE_HTTPS_WEB
-      clusters:
-        - name: local2
-          trust_zone: tz2
-          kubernetes_context: kind-local2
-          trust_provider:
-            kind: kubernetes
-          profile: kubernetes
-          external_server: false
+clusters:
+    - name: local1
+      trust_zone: tz1
```

Comment:
Not specific feedback to this diff itself, but can we say strictly that a Cluster will only ever have a single Trust Zone associated with it? Good if we can and I think we can, I'm just making sure there's not some potential overlap of zones case I'm missing

Reply:
I think it's a reasonable assumption for now.